### PR TITLE
Add setup instructions in readme file

### PR DESCRIPTION
**Pull request by inovizz**
- Make changes in README file for setup instructions
- Change the command in build.sh to run node server from Jenkins.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
-## Building and the frontend
 
-1. First run `truffle compile`, then run `truffle migrate` to deploy the contracts onto your network of choice (default "development").
-1. Then run `npm start` to build the app and serve it on http://localhost:8080
+## Quickstart
+
+1. Clone the repo
+2. install testrpc  `npm install -g ethereumjs-testrpc` and truffle `npm install truffle@3.2.5 -g`
+3. Go to project dir and run `npm install`
+3. Run `truffle compile`, then run `truffle migrate` to deploy the contracts onto your network of choice (default "development").
```
Comment (pawansingh126) [inline README.md]: - need to add step to run "testrpc" before migrate.
- numbering
- optional step to run tests via "truffle test"

**Comment by neeldhwaj** [comment]
LGTM!

File: README.md (hunk 1 above)
Comment (inovizz) [reply README.md]: Made the necessary changes, thanks.

**Comment by neeldhwaj** [comment]
LGTM!

**Pull request merged**